packages/rest/src/lib/CDN.ts

```diff
@@ -119,12 +119,12 @@ export class CDN {
 	}
 
 	/**
-	 * Generates the default avatar URL for a discriminator.
+	 * Generates the default avatar URL for an user id.
 	 *
-	 * @param discriminator - The discriminator modulo 5
+	 * @param userId - The user id modulo 5
```

Comment:
Well this ain't accurate, since the formula is `(userId >> 22) % 5`. You should either find a better name for it or maybe document both variants (legacy discrim and userid)

Reply:
I like Jira's suggestion